app/Http/Controllers/AdminDash.php

```diff
@@ -1495,6 +1495,11 @@ public function setEventPositionPreset(Request $request, $id) {
         return redirect()->back()->with('success', 'The position preset has been added successfully');
     }
 
+    public function sendEventReminder($id) {
+        Artisan::call('Event:SendEventReminder ' . $id);
+        return redirect()->back()->with('success', 'Event reminder sent');
+    }
```

Comment:
Would it be useful to keep track of whether or not a reminder has been sent (could just be a boolean in the DB) that then warns the user before sending a reminder if one has already been sent? I feel like it would be really easy for 2 people to accidentally send duplicate reminders on accident

Reply:
There is already protection that prevents a registrant from getting two reminder emails. Ref line 58, if ($r->reminder != 1) {

I prefer that we keep it this way... allows the EC to trigger a reminder to people that sign-up late that may have missed the first reminder.

Reply:
Gotcha - I didn't realize that was there - that solves my concern!